x-pack/plugins/session_view/public/components/session_view/index.tsx

```diff
@@ -92,127 +93,131 @@ export const SessionView = ({ sessionEntityId, height, jumpToEvent }: SessionVie
-                    isFetching={isFetching}
-                    hasPreviousPage={hasPreviousPage}
-                    hasNextPage={hasNextPage}
-                    fetchNextPage={fetchNextPage}
-                    fetchPreviousPage={fetchPreviousPage}
-                    setSearchResults={setSearchResults}
-                    timeStampOn={displayOptions.timestamp}
-                    verboseModeOn={displayOptions.verboseMode}
+                {error && (
+                  <EuiEmptyPrompt
+                    iconType="alert"
+                    color="danger"
+                    title={
+                      <h2>
+                        <FormattedMessage
+                          id="xpack.sessionView.errorHeading"
+                          defaultMessage="Error loading Session View"
+                        />
+                      </h2>
+                    }
+                    body={
+                      <p>
+                        <FormattedMessage
+                          id="xpack.sessionView.errorMessage"
+                          defaultMessage="There was an error loading the Session View."
+                        />
+                      </p>
+                    }
                   />
-                </div>
-              )}
-            </EuiResizablePanel>
+                )}
 
-            {renderDetails ? (
-              <>
-                <EuiResizableButton />
-                <EuiResizablePanel
-                  id="session-detail-panel"
-                  initialSize={30}
-                  minSize="200px"
-                  paddingSize="none"
-                  css={styles.detailPanel}
-                >
-                  <SessionViewDetailPanel
-                    selectedProcess={selectedProcess}
-                    onProcessSelected={onProcessSelected}
-                  />
-                </EuiResizablePanel>
-              </>
-            ) : (
-              <>
-                {/* Returning an empty element here (instead of false) to avoid a bug in EuiResizableContainer */}
-              </>
-            )}
-          </>
-        )}
-      </EuiResizableContainer>
+                {hasData && (
+                  <div css={styles.processTree}>
+                    <ProcessTree
+                      sessionEntityId={sessionEntityId}
+                      data={data.pages}
+                      searchQuery={searchQuery}
+                      selectedProcess={selectedProcess}
+                      onProcessSelected={onProcessSelected}
+                      jumpToEvent={jumpToEvent}
+                      isFetching={isFetching}
+                      hasPreviousPage={hasPreviousPage}
+                      hasNextPage={hasNextPage}
+                      fetchNextPage={fetchNextPage}
+                      fetchPreviousPage={fetchPreviousPage}
+                      setSearchResults={setSearchResults}
+                      timeStampOn={displayOptions.timestamp}
```

Comment:
note: think on the future we can replace to pass only one property `displayOptions={displayOptions}` to suit all display options (don't need to do it right now)